### PR TITLE
Update ketcher to 2.11

### PR DESCRIPTION
**Pull request by bjonnh**
This requires a recent streamlit as it needs a more recent react.

It solves a lot of issues with aromatic molecules (where the SMILES produced were invalid)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ classifiers = [
 ]
 
 dependencies = [
-    "streamlit>=1.0.0",
+    "streamlit>=1.25.0",
```
[the remark this answers is not in the document]
Comment (bjonnh-work) [reply pyproject.toml:24]: I had issues building with older versions but we can try.

Also yes it looks like the images I generate on my machine are different from the ones in the PR... (still those toolbars)

**Comment by bjonnh** [comment]
I have the actions running on my side: https://github.com/bjonnh/streamlit-ketcher/actions
So you can check for flakyness across accounts too.

**Comment by mik-laj** [comment]
Thanks for your contributions.

I managed to fix the E2E tests, but I'll have to work on the process to make them easier to maintain.

**Comment by bjonnh** [comment]
Thank you!

**Pull request merged**